README.md

```diff
@@ -295,35 +360,35 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
 This section is a primer on how companies raise funding and grow, as this is critical to understanding the value of a company and what equity in a company is worth.
 
 - The stage of a startup is largely reflected in how much funding it has raised. Very roughly, [typical levels](http://blog.eladgil.com/2011/03/how-funding-rounds-differ-seed-series.html) are:
-    - **Bootstrapped**: No funding. Founders are figuring out what to build, or they're starting to build with their own time and resources.
+    - **Bootstrapped**: No funding. Founders are figuring out what to build, or they’re starting to build with their own time and resources.
     - **Series Seed** ($250K to $2 million): Figuring out the product and market.
     - **Series A** ($2 to $15 million): Scaling the product and making the business model work.
     - **Series B** (tens of millions): Scaling the business.
     - **Series C, D, E,** etc. (tens to hundreds of millions): Continued scaling of the business.
 - 🔸 Most startups don’t get far. Very roughly, if you [look at angel investments](http://codingvc.com/valuing-employee-options/), **more than half** of investments fail, **one in 3** are small successes (1X to 5X returns), **one in 8** are big successes (5X to 30x), and **one in 20** are huge successes (30X+).
-- 🔸 Each stage reflects the removal of risk and increased [**dilution**](http://www.investopedia.com/terms/d/dilution.asp). For this reason, the equity team members get is higher in the earlier stages (starting with founders) and increasingly lower as a company matures. (See the picture above.)
+- 🔸 Each stage reflects the removal of risk and increased dilution. For this reason, the equity team members get is higher in the earlier stages (starting with founders) and increasingly lower as a company matures. (See the picture above.)
 - 🔹 It is critical to understand risk and dilution to know the possible future value of equity. [This article](http://codingvc.com/valuing-employee-options/) from Leo Polovets, a partner at Susa Ventures, gives a good overview.
-- 🔹 If you’re talking with a startup, there are a lot of questions to ask in order to assess the state of the company's business. Startups are understandably careful about sharing financial information, so you may not get full answers to all of these, but you should at least ask:
+- 🔹 If you’re talking with a startup, there are a lot of questions to ask in order to assess the state of the company’s business. Startups are understandably careful about sharing financial information, so you may not get full answers to all of these, but you should at least ask:
```

Comment:
This is the best list of questions I've seen: https://blog.wealthfront.com/stock-options-14-crucial-questions/